.github/workflows/gutenberg-packages-update.yml

```diff
@@ -159,19 +160,37 @@ jobs:
         if: steps.packages.outputs.list != 0
         run: |
           git add --all .
-          git commit -m "Update Gutenberg package dependencies"
-          git push origin "$HEAD_BRANCH"
+          git commit -m "Update WordPress NPM packages based on Gutenberg release v$VERSION" -m "Signed-off-by: $(git config user.name) <$(git config user.email)>"
+          git push -u origin "$HEAD_BRANCH"
         env:
+          VERSION: ${{ needs.check-gutenberg-release.outputs.latest-version }}
           HEAD_BRANCH: ${{ steps.branches.outputs.head }}
 
-      - name: Create pull request
+      - name: Draft a pull request
+        id: pr-create
         if: steps.packages.outputs.list != 0 && steps.remote-branch.outputs.exists == 0
         run: |
-          git push -u origin "$HEAD_BRANCH"
-          PR_URL=$(gh pr create --base "$BASE_BRANCH" --title "Update Gutenberg packages after v$VERSION release" --body "" --label dependencies | grep https://)
-          gh pr merge --auto --merge "$PR_URL"
+          PR_BODY=$(node bin/gutenberg-packages-update-pr-body.js "${{ steps.packages.outputs.list }}")
+          PR_URL=$(gh pr create --draft --base "$BASE_BRANCH" --title "Update Gutenberg packages after v$VERSION release" --body "$PR_BODY" --label dependencies | grep https://)
+          echo "::set-output name=pr-url::$PR_URL"
         env:
           VERSION: ${{ needs.check-gutenberg-release.outputs.latest-version }}
           GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
           BASE_BRANCH: ${{ steps.branches.outputs.base }}
           HEAD_BRANCH: ${{ steps.branches.outputs.head }}
+
+      # Make PR ready using personal access token to allow workflow to be triggered.
+      - name: Make PR ready
+        if: steps.pr-create.outputs.pr-url != ''
+        run: gh pr ready "$PR_URL"
+        env:
+          GITHUB_TOKEN: ${{ secrets.GH_PAT }}
+          PR_URL: ${{ steps.pr-create.outputs.pr-url }}
```

Comment:
So when this runs, the `auto-merge` workflow will then run?

Reply:
Yes. When we make this PR ready with someone's `personal access token` then it will run the `Build Test & Measure` workflow. When all checks will pass it will trigger the `auto-merge`.